src/Bindings/LuaState.h

```diff
@@ -35,6 +35,9 @@ extern "C"
 #include "PluginManager.h"
 #include "LuaState_Typedefs.inc"
 
+#undef TOLUA_TEMPLATE_BIND
+#include "tolua++/include/tolua++.h"
+
```

Comment:
Revert this change, move the inclusion back into the cpp file. We don't want to include the tolua library into each compilation unit including LuaState.h